instrumentation/lettuce/lettuce-4.0/javaagent/src/test/groovy/LettuceAsyncClientTest.groovy

```diff
@@ -71,18 +70,12 @@ class LettuceAsyncClientTest extends AgentInstrumentationSpecification {
     dbUriNonExistent = "redis://" + dbAddrNonExistent
     embeddedDbUri = "redis://" + dbAddr
 
-    redisServer = RedisServer.builder()
-    // bind to localhost to avoid firewall popup
-      .setting("bind " + HOST)
-    // set max memory to avoid problems in CI
-      .setting("maxmemory 128M")
-      .port(port).build()
+    redisServer = redisServer.withFixedExposedPort(port, 6379)
```

Comment:
Do we need `withFixedExposedPort` or can we use normal testcontainers to get a random port?

Reply:
Some test classes have tests that kill this container. Thus they restart that in `setup`. Which means that normal container with a random port will get a new port for each test. This can be done but requires more changes to setups. I decided against that, at least in this PR